tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {ansible2.1,ansible2.2}-{initial-members}
+envlist = {ansible2.1}-{xenial-conf-tests,xenial-mon-osd,centos7-mon-osd}
```

Comment:
Yeah, I thought of that as well. The ``envlist`` is just used with you don't pass the ``-e`` flag though, we can be descriptive with our local testing and CI testing using ``-e`` for now.